lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -115,9 +117,13 @@ defmodule ExUnit.CaptureLog do
           {:DOWN, ^ref, :process, ^parent, _reason} -> :ok
           {:gen_event_EXIT, {Console, ^pid}, _reason} -> :ok
         end
-      other ->
-        :proc_lib.init_ack(other)
+      {:EXIT, reason} ->
+        :proc_lib.init_ack({:error, reason})
+      {:error, reason} ->
```

Comment:
Is this result possible from `add_sup_handler`? I think we need just a catch all here.

Reply:
@fishcakez Sorry I didn't see this comment before. You're right, or at least `add_sup_handler` does not document this kind of return, will you fix it on `master` Or do you want me to change and open another PR?